### PR TITLE
adding ruby2.3

### PR DESCRIPTION
**Pull request by ferdinandrosario**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ rvm:
   - '2.0'
   - '2.1'
   - '2.2'
+  - 2.3.0
```
Comment (byroot) [inline .travis.yml]: Just 2.3 please so it keeps updated.

**Comment by ferdinandrosario** [comment]
Travis have no binaries...

Requested binary installation but no rubies are available to download, consider skipping --binary flag.
Gemset '' does not exist, 'rvm ruby-2.3 do rvm gemset create ' first, or append '--create'.

**Comment by byroot** [comment]
Grumble. Ok sorry then. Let's go back to `2.3.0`

**Comment by ferdinandrosario** [comment]
:smile:

**Pull request closed without merging**